libs/common/src/helpers/date-validation.ts

```diff
@@ -214,3 +214,17 @@ export const applyStandardValidationAndParseDateInput = (inputtedDate: DayMonthY
     parsedDate,
   };
 };
+
+/**
+ * checks if provided value is a date
+ * @param date - provided date
+ * @returns if the provided date is an instance of Date
+ */
+export const isDate = (date: any): boolean => date instanceof Date;
+
+/**
+ * checks if provided value is a date and is not invalid
+ * @param date - provided date
+ * @returns if the provided date is an instance of Date and is not invalid
+ */
+export const isValidDate = (date: any): boolean => date instanceof Date && !Number.isNaN(date.getTime());
```

Comment:
Can we remove `isDate` and use `isValidDate` throughout? Feels strange to have both of these functions.

Currently, `isDate` could return an invalid date

Reply:
removed